### ISSUE TITLE
Слишком длинные паузы на запятой.

### ISSUE THREAD

**Issue by mike2003** [RHVoice/RHVoice#385]
При прослушивании текста на лету, создается стойкое ощущение, что паузы на запятой слишком длинные, даже длиннее точек. Возникает дискомфорт из за таких спотыканий внутри предложения. Использую Балаболку для чтения.
Как это можно корректировать? Поменять их между собой? Не знаю куда копать.
В настройках Балаболки есть паузы для точек и абзацев, но они не влияют.

**Comment by mike2003** [comment]
Вот пример
![zDudi1U](https://user-images.githubusercontent.com/11754198/128742293-d69c5c1c-d311-485a-9dda-0b65f858e163.jpg)

**Comment by alex19EP** [comment]
вы используете RHVoice как sapi модуль?
можете повторить проблему не только с балаболкой?

пока мне кажется, что проблема с ней.

**Comment by mike2003** [comment]
Да, sapi 5 для чтения текста на лету хочу. С чем еще можно проверить кроме балаболки? Я немного не в теме )

**Comment by mike2003** [comment]
Никакой помощи не будет?

**Comment by alex19EP** [comment]
> Да, sapi 5 для чтения текста на лету хочу. С чем еще можно проверить кроме балаболки? Я немного не в теме )

я к сожалению тоже не очень то и в теме. возможно есть какие-то другие читалки под windows.
или если вы используете скринридер, то возможно можете прочитать тотже текст им, а затем понять действительно ли проблема в RHVoice.
как я уже говорил - на данный момент я очень сомневаюсь в этом.

**Comment by mike2003** [comment]
Проверил в другой программе TextAloud 4 - то же самое. Странно, что такой старый движок не умеет со знаками препинания работать. Как же такое пропустили?
